CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 0.3.0
+
+⚠️ BREAKING ⚠️
```

Comment:
We could avoid this being breaking since these are optional named properties, but I don't think that's necessary for this stage in the project?

Reply:
Oh, yeah, that's a good point. But, yeah, I think we can do it this way for pre 1.0 releases.